h5py/tests/test_vds/test_highlevel_vds.py

```diff
@@ -338,5 +339,64 @@ def test_index_layout(self):
     def tearDown(self):
         shutil.rmtree(self.tmpdir)
 
+@ut.skipUnless(vds_support,
+               'VDS requires HDF5 >= 1.9.233')
+class RelativeLinkTestCase(ut.TestCase):
+
+    def setUp(self):
+        self.tmpdir = tempfile.mkdtemp()
```

Comment:
h5py is using pytest now - we haven't converted all the old tests, but you're welcome to write new tests as functions with fixtures if you like.

Reply:
should we convert all unittest to pytest?  if so, I'd like to have a try.

Reply:
I think we should leave them - converting them all is a lot of work, and even if you're volunteering to do that, someone else has to review a lot of changes. As we've seen, even seemingly simple changes can break things. And h5py wouldn't really be any better for it - the existing unittest-style tests do what they're supposed to.

Other maintainers may disagree, but unless one of them makes that case, please don't start converting them. If you're looking for something else to do, we tagged a few issues [good first issue](https://github.com/h5py/h5py/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) for the sprint last weekend.

Reply:
Ok, got it 😃